documentation/docs/specs/did/iota_did_method_spec.md

```diff
@@ -47,257 +68,146 @@ A DID that uses this method MUST begin with the following prefix: `did:iota`. Fo
+`State Metadata` in the Alias Output which reflects in updating the DID Document.
 
-In order to create a valid Integration DID message, the following steps are to be performed:
-1. Create a new DID Document that contains all the new target state. This SHOULD include the new desired changes and all the changes inside the previous Diff Chain, otherwise these changes are lost!
-2. Retrieve the IOTA MessageId from the previous Integration DID message and a keypair for signing the DID Document.
-3. Set the `previousMessageId` field to the IOTA MessageId value.
-4. Create a `proof` object referencing the public key and signature suite from a verification method in the DID Document able to sign updates as per the [Signing Keys](#signing-keys) section.
-5. Create a cryptographic signature using the same keypair and add the result to the appropriate field(s) inside the `proof` object.
+1. Create the content of the updated DID Document.
+2. Use placeholder `did:0:0` instead of the actual DID in the document.
+3. Create the payload and the headers as described in the [Anatomy of the State Metadata](#anatomy-of-the-state-metadata).
+4. Create a new Alias Output with the same `Alias ID` as the current output.
+5. Increment the `State Index`.
+6. Set the state controller and the governor.
+7. Publish a new transaction with the current Alias Output as input and the newly created one as output.
 
-#### Creating a Diff DID message
+Note, if the updated document requires more coins to cover the storage deposit, another input will be required to cover that.
 
-A Differentiation DID message is restricted in its usage. It may not update any signing keys that are used in the Diff chain. If this is desired, it is REQUIRED to use an Integration DID message. If the current Diff chain becomes too long (currently RECOMMENDED to end at a length of 100), it is RECOMMENDED to use a single Integration DID message to reset its length.
+### Delete
 
-In order to create a valid Integration DID message, the following steps are to be performed:
-1. Create and serialize a `diff` JSON object that contains all the changes. 
-2. Set the `did` field to the DID value that this update applies to.
-3. Retrieve the IOTA MessageId from the previous Diff chain message, or Integration DID message if this message is the first in the Diff chain. In addition, retrieve a keypair for signing the DID Document.
-4. Set the `previousMessageId` field to the IOTA MessageId value.
-5. Create a `proof` object referencing the public key and signature suite from a verification method in the DID Document able to sign updates as per the [Signing Keys](#signing-keys) section.
-6. Create a cryptographic signature using the same keypair and add the result to the appropriate field(s) inside the `proof` object.
+#### Deactivate
+Temporarily deactivating a DID can be done by deleting the content of the `State Meadata` in the Alias Output. This will render the DID unresolvable.
```

Comment:
It can still be resolved, but it will yield a deactivated document.

```suggestion
Temporarily deactivating a DID can be done by deleting the content of the `State Meadata` in the Alias Output.
```